test/aqua/test_optimizer_aqqd.py

```diff
@@ -42,10 +42,15 @@ def setUp(self):
     def test_aqgd(self):
         """ test AQGD optimizer by using it """
 
-        result = VQE(self.qubit_op,
-                     RealAmplitudes(),
-                     AQGD(momentum=0.0)).run(
-                         QuantumInstance(BasicAer.get_backend('statevector_simulator'),
-                                         seed_simulator=aqua_globals.random_seed,
-                                         seed_transpiler=aqua_globals.random_seed))
-        self.assertAlmostEqual(result.eigenvalue.real, -1.857275, places=5)
+        q_instance = QuantumInstance(BasicAer.get_backend('statevector_simulator'),
```

Comment:
Can we please split this into 3 separate tests so if there is a failure its easier to pinpoint it to a specific test aspect

Reply:
Done.